modules/code-editor/src/views/full/FileNavigator.tsx

```diff
@@ -138,6 +193,21 @@ class FileNavigator extends React.Component<Props, State> {
     e.preventDefault()
 
     if (!node.nodeData || this.props.disableContextMenu) {
+      if (this.state.selectedFilesCount > 0){
+        const { selectedFilesList } = this.state
+        ContextMenu.show(
+          <Menu>
+            <MenuItem
+              id="btn-paste"
+              icon="clipboard"
+              text={lang.tr('module.code-editor.navigator.paste')}
+              onClick={() => this.props.bulkMoveFiles(Object.values(selectedFilesList), node.id as string)}
```

Comment:
check if `as string` is necessary here since `node` is typed

Reply:
As of `src/components/tree/treeNode.tsx` on `node_modules`
`node.id` is of type `string | number`
![image](https://user-images.githubusercontent.com/13097764/129971585-8f352198-af96-42cb-8668-40092bb81582.png)